### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by nhakmiller**
## Background

Snyk detected some out of date python dependencies with vulnerabilities and available updates. Applying those updates here.

## Changes

- Re-installed python dependencies

## Testing

- Unit tests

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,17 +2,17 @@ arrow==0.17.0
 astroid==2.4.2
 attrs==20.3.0
 aws-lambda-builders==1.1.0
-aws-sam-cli==1.11.0
-aws-sam-translator==1.30.1
-bandit==1.6.2
+aws-sam-cli==1.15.0
```
Comment (austinbyers) [inline requirements.txt:5]: I'll be removing this one soon!

**Review by austinbyers — APPROVED** [review]
Do the libraries in the Python layer need to be updated as well?

**Comment by nhakmiller** [comment]
> Do the libraries in the Python layer need to be updated as well?

Might be a good idea, is this just updating their versions in `panther_config.yml`? If so I can add that.

**Comment by austinbyers** [comment]
> Might be a good idea, is this just updating their versions in panther_config.yml? If so I can add that.

Yes, as well as the [associated documentation](https://docs.runpanther.io/user-guide/analysis/run-time) (once it syncs to enterprise)

**Comment by nhakmiller** [comment]
> > Might be a good idea, is this just updating their versions in panther_config.yml? If so I can add that.
> 
> Yes, as well as the [associated documentation](https://docs.runpanther.io/user-guide/analysis/run-time) (once it syncs to enterprise)

Strangely enough they appear to be at the same version as the packages in `requirements.txt` even after the updates, I guess these packages don't change too often.

**Pull request merged**